### PR TITLE
tool: make the use of parameters unambiguous

### PR DESCRIPTION
**Pull request by simon-rock**
Signed-off-by: simon gao <simon29rock@gmail.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/tools/ceph_monstore_tool.cc [hunk 1]
```diff
@@ -877,7 +875,7 @@ int main(int argc, char **argv) {
       goto done;
     }
 
-    if (readable) {
+    if (op_vm.count("help")) {
```
Comment (tchaikov) [inline src/tools/ceph_monstore_tool.cc]: s/help/readable/ ?

Comment (simon-rock) [reply src/tools/ceph_monstore_tool.cc]: sorry. I make a mistake. I have repushed it.

**Comment by simon-rock** [comment]
> i major concern is that, this change is not backward compatible. if user have scripts using this tool, it will bail out with follow error message
> 
> ```
> error: too many positional options have been specified on the command line
> ```
yes. I have checked tools that relies on "ceph_monstore_tool" in the project. They 
didn't use this parameter.
I used this tool a long time ago. But I just spent some time studying how to use it to output mdsmap.

**Comment by simon-rock** [comment]
@tchaikov  readable is ont a positional options. I change it to bool options.

**Comment by tchaikov** [comment]
@simon-rock if you want to resurrect this PR. please update the commit the commit message with the rationale behind the change and add a "`Signed-off-by`" line at the end of your commit message? "git commit -s" will do the trick for you. see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work

**Comment by simon-rock** [comment]
@tchaikov 
I only changed the usage of parameter, readable.
--readable=0    ==> --readable

I didn't change it to a position parameter, which would probably cause
problems I think.
the error, error: too many positional options have been specified on
the command line, is caused by too many positional parameters.

**Comment by simon-rock** [comment]
@tchaikov I have update the commit on the branch, but the new commit cannot be synchronized because the pr has been closed, I think. can you reopen this pr?

**Comment by tchaikov** [comment]
@simon-rock you will have to create a new PR from your updated branch.

**Comment by simon-rock** [comment]
@tchaikov I have update the commit. but the new commit cannot be synchronized. I think the reason maybe the pr is closed. can you reopen this pr.

> @simon-rock you will have to create a new PR from your updated branch.

@tchaikov  ok. thx. the new pr is https://github.com/ceph/ceph/pull/32265

**Comment by simon-rock** [comment]
> @simon-rock you will have to create a new PR from your updated branch.

@tchaikov ok. thx. the new pr is #32265

**Pull request closed without merging**